static/js/app.ts

```diff
@@ -18,6 +18,7 @@ import { HedyEditor, EditorType } from './editor';
 import { HedyAceEditorCreator } from './ace-editor';
 import { stopDebug } from "./debugging";
 
+export let theGlobalDebugger: any;
```

Comment:
Would it make sense (not in this PR but in the future!) to also make an interface for the debugger like we do for the editor?

Reply:
Yes! I'm planning on two things: leave the debugger in JavaScript and write an vendor interface like we do for some other Skulpt code we execute . Or, convert it entilery to TypeScript.

The interface we use for Skulpt is here ((https://github.com/hedyorg/hedy/blob/4c4d24ee0322f37205f0bed2c1a5e44e7f4a1d39/static/js/vendor.d.ts#L4-L40))